packages/storybook/mock/next-router.ts

```diff
@@ -1,7 +1,8 @@
 import { fn } from '@storybook/test';
 
-export const useRouter = fn().mockReturnValue({
+export const useRouter = () => ({
```

Comment:
Would there be any difference in using this over the previous one? Asking just so we can have clarity and which one to go for on the next ones.

Reply:
When using "mockReturnValue" the router was erroring with a "Cannot destructure property 'query' of 'useRouter(...)' as it is undefined.". We didn't really figure out why it didn't work, but @ilasw suggested we just remove the mock because the router does not need to be tested at this point, so we can just add a skeleton so that it doesn't cause errors.